main.rb

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env ruby
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/MethodLength: Method has too many lines
```

Comment:
- [ ] Please try to not disable linters because they're there to help you write a better code.

Reply:
- [ ] Fix method length issue and remove the corresponding disable method